cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/casLoginNotAllowedView.jsp

```diff
@@ -18,14 +18,31 @@
     under the License.
 
 --%>
-<jsp:directive.include file="includes/top.jsp" />
 
-  <div id="msg" class="errors">
-    <spring:eval var="osfResendConfirmationUrl" expression="@casProperties.getProperty('osf.resendConfirmation.url')" />
-    <h2><spring:message code="screen.loginnotallowed.heading" /></h2>
-    <p><spring:message code="screen.loginnotallowed.message" arguments="${osfResendConfirmationUrl}" /></p>
-  </div>
+<%-- OSF Login Exception Page -- Account Not Confirmed --%>
 
-<c:set var="alternativeBottomLogin" value="true"/>
+<jsp:directive.include file="includes/top.jsp"/>
 
-<jsp:directive.include file="includes/bottom.jsp" />
+<div id="msg" class="errors">
+
+    <h2><spring:message code="screen.loginnotallowed.heading"/></h2>
+    <p><spring:message code="screen.loginnotallowed.message"/></p>
+
+    <hr/>
+    <br/>
+    <section class="row">
+        <spring:eval var="osfResendConfirmationUrl"
+                     expression="@casProperties.getProperty('osf.resendConfirmation.url')"/>
+        <a id="alt-login-inst" class="btn-alt-login" href="${osfResendConfirmationUrl}">
+            <img class="osf-alt-logo" src="../images/osf-alt-logo.png">
+            <span class="label-login"><spring:message code="screen.loginnotallowed.button.resendConfirmation"/></span>
+        </a>
+    </section>
+</div>
+
+<c:set var="linkSignIn" value="true"/>
```

Comment:
For the account not confirmed page, show "Sign In", "Create Account" and "Back to OSF".